<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-12-17T23:21:45 IST" name="Suite" finished-at="2024-12-17T23:21:45 IST" duration-ms="267">
    <groups>
      <group name="smoke">
        <method signature="TC002_CalculatorTest.calculateCarEmi()[pri:0, instance:testCases.TC002_CalculatorTest@55141def]" name="calculateCarEmi" class="testCases.TC002_CalculatorTest"/>
        <method signature="TC002_CalculatorTest.enterFieldAmount(java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:testCases.TC002_CalculatorTest@55141def]" name="enterFieldAmount" class="testCases.TC002_CalculatorTest"/>
      </group> <!-- smoke -->
      <group name="sanity">
        <method signature="TC002_CalculatorTest.calculateCarEmi()[pri:0, instance:testCases.TC002_CalculatorTest@55141def]" name="calculateCarEmi" class="testCases.TC002_CalculatorTest"/>
      </group> <!-- sanity -->
    </groups>
    <test started-at="2024-12-17T23:21:45 IST" name="Test" finished-at="2024-12-17T23:21:45 IST" duration-ms="267">
      <class name="testCases.TC002_CalculatorTest">
        <test-method signature="calculateCarEmi()[pri:0, instance:testCases.TC002_CalculatorTest@55141def]" started-at="2024-12-17T23:21:45 IST" name="calculateCarEmi" finished-at="2024-12-17T23:21:45 IST" duration-ms="92" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.openqa.selenium.SearchContext.findElement(org.openqa.selenium.By)" because "this.searchContext" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.SearchContext.findElement(org.openqa.selenium.By)" because "this.searchContext" is null
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at jdk.proxy2/jdk.proxy2.$Proxy12.click(Unknown Source)
at pageObjec.HomePage.clickcalculatorLink(HomePage.java:20)
at testCases.TC002_CalculatorTest.calculateCarEmi(TC002_CalculatorTest.java:18)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- calculateCarEmi -->
        <test-method signature="enterFieldAmount(java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:testCases.TC002_CalculatorTest@55141def]" started-at="2024-12-17T23:21:45 IST" name="enterFieldAmount" finished-at="2024-12-17T23:21:45 IST" depends-on-methods="testCases.TC002_CalculatorTest.calculateCarEmi" duration-ms="1" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method testCases.TC002_CalculatorTest.enterFieldAmount() on instance testCases.TC002_CalculatorTest@55141def depends on not successfully finished methods 
[testCases.TC002_CalculatorTest.calculateCarEmi() on instance testCases.TC002_CalculatorTest@55141def]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method testCases.TC002_CalculatorTest.enterFieldAmount() on instance testCases.TC002_CalculatorTest@55141def depends on not successfully finished methods 
[testCases.TC002_CalculatorTest.calculateCarEmi() on instance testCases.TC002_CalculatorTest@55141def]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:159)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterFieldAmount -->
      </class> <!-- testCases.TC002_CalculatorTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
